compiler-cli/src/dependencies.rs

```diff
@@ -24,6 +24,84 @@ use crate::{
     http::HttpClient,
 };
 
+pub fn list() -> Result<()> {
+    let config = crate::config::root_config()?;
+    let manifest = read_manifest_from_disc()?;
+    list_manifest_packages(std::io::stdout(), config.name, manifest)
```

Comment:
I noticed there's also [`cli::stdout_buffer_writer`](https://github.com/gleam-lang/gleam/blob/235eb17ff473cef461cdca0478302e23ba610aea/compiler-cli/src/cli.rs#L134-L137). Would it be best to write to that instead? 😄

Reply:
I think this is good as we're not using any colourful printing features